sdk/resources/Microsoft.Azure.Management.Resource/tests/Microsoft.Azure.Management.Resource.Tests.csproj

```diff
@@ -38,6 +38,9 @@
     <None Update="ScenarioTests\management_group_level_template.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="ScenarioTests\tenant_level_template.json">
```

Comment:
Could we use `Update="ScenarioTests\**\*.json"` instead to avoid list all json files here?

Reply:
updated.